brave/src/main/java/brave/handler/MutableSpan.java

```diff
@@ -90,6 +91,9 @@ public MutableSpan() {
   /** @since 5.12 */
   public MutableSpan(MutableSpan toCopy) {
     if (toCopy == null) throw new NullPointerException("toCopy == null");
+    traceId = toCopy.traceId;
+    parentId = toCopy.parentId;
```

Comment:
Missing `localRootId`

Reply:
ooohh I can use the copy constructor in the existing tests! thanks for the keen eye meanwhile!